src/discussions/posts/post-actions-bar/PostActionsBar.jsx

```diff
@@ -25,30 +26,36 @@ function PostActionsBar({
   const handleCloseInContext = () => {
     postMessageToParent('learning.events.sidebar.close');
   };
+
+  const loadingStatus = useSelector(selectconfigLoadingStatus);
   const blackoutDateRange = useSelector(selectBlackoutDate);
```

Comment:
nits: avoid unnecessary newlines and please move hooks at the start of the component.